tests/airgap/airgap_provisioning_test.go

```diff
@@ -159,12 +158,8 @@ func (a *TfpAirgapProvisioningTestSuite) TestTfpAirgapUpgrading() {
 			defer cleanup.Cleanup(a.T(), a.terraformOptions, keyPath)
 
 			clusterIDs := provisioning.Provision(a.T(), a.client, a.rancherConfig, &terraformConfig, &terratestConfig, testUser, testPassword, clusterName, poolName, a.terraformOptions, nil)
-			provisioning.VerifyClustersState(a.T(), a.client, clusterIDs)
```

Comment:
did you mean to remove the `VerifyClusterState` function here? 🙂  -- just noticing this is the only place where that was removed as well